### ISSUE TITLE
feat(docs): sortable something

### ISSUE THREAD

**Issue by trueVitality** [valor-software/ngx-bootstrap#2250]
In bs-sortable teg need add -> [ngModelOptions]="{standalone: true}".
And all work in children modules. PLZ FIX ON OFFICIAL SITE!
I wasted 4 hours! Thx for good DOC!

**Comment by valorkin** [comment]
@trueVitality can you explain in plain english (russian) what you just said?

**Comment by valorkin** [comment]
PS sortable is an experimental feature

**Comment by IlyaSurmay** [comment]
Closed until author comes back with better description. :)
